src/@types/prisma.ts

```diff
@@ -0,0 +1,57 @@
+export interface IPrismaSession {
+  id: string;
+  sid: string;
+  data: string | null;
+  expires: Date;
+}
+
+interface ICreatePrismaSession extends IPrismaSession {
+  data: string;
+}
+
+interface IFindOneArgs {
+  where: {
+    sid: string;
+  };
+  select?: {
+    expires?: boolean;
+    sid?: boolean;
```

Comment:
These were simply not implemented as they didn't need to be.
The idea with the interface was to specify everything we require `PrismaClient` to do but not necessarily all it can do.
If we write code in the future that requires those to be selected they can be added here easily enough and by making this less strict it protects us from non breaking API changes in Prisma if they decide to remote the `id` from select for some reason for example.

Reply:
>The idea with the interface was to specify everything we require PrismaClient to do but not necessarily all it can do.
Ok.